smartapps/curb/curb-bridge.src/curb-bridge.groovy

```diff
@@ -0,0 +1,605 @@
+
+  <head>
+      <meta name="viewport" content="width=640">
+      <title>Curb & SmartThings connection</title>
+      <style type="text/css">
+          @font-face {
+              font-family: 'Swiss 721 W01 Thin';
+              src: url('https://s3.amazonaws.com/smartapp-icons/Partner/fonts/swiss-721-thin-webfont.eot');
+              src: url('https://s3.amazonaws.com/smartapp-icons/Partner/fonts/swiss-721-thin-webfont.eot?#iefix') format('embedded-opentype'),
+              url('https://s3.amazonaws.com/smartapp-icons/Partner/fonts/swiss-721-thin-webfont.woff') format('woff'),
+              url('https://s3.amazonaws.com/smartapp-icons/Partner/fonts/swiss-721-thin-webfont.ttf') format('truetype'),
+              url('https://s3.amazonaws.com/smartapp-icons/Partner/fonts/swiss-721-thin-webfont.svg#swis721_th_btthin') format('svg');
+              font-weight: normal;
+              font-style: normal;
+          }
+
+          @font-face {
+              font-family: 'Swiss 721 W01 Light';
+              src: url('https://s3.amazonaws.com/smartapp-icons/Partner/fonts/swiss-721-light-webfont.eot');
+              src: url('https://s3.amazonaws.com/smartapp-icons/Partner/fonts/swiss-721-light-webfont.eot?#iefix') format('embedded-opentype'),
+              url('https://s3.amazonaws.com/smartapp-icons/Partner/fonts/swiss-721-light-webfont.woff') format('woff'),
+              url('https://s3.amazonaws.com/smartapp-icons/Partner/fonts/swiss-721-light-webfont.ttf') format('truetype'),
+              url('https://s3.amazonaws.com/smartapp-icons/Partner/fonts/swiss-721-light-webfont.svg#swis721_lt_btlight') format('svg');
+              font-weight: normal;
+              font-style: normal;
+          }
+
+          .container {
+              width: 90%;
+              padding: 4%;
+              text-align: center;
+          }
+
+          img {
+              vertical-align: middle;
+          }
+
+          p {
+              font-size: 2.2em;
+              font-family: 'Swiss 721 W01 Thin';
+              text-align: center;
+              color: #666666;
+              padding: 0 40px;
+              margin-bottom: 0;
+          }
+
+          span {
+              font-family: 'Swiss 721 W01 Light';
+          }
+      </style>
+  </head>
+
+  <body>
+      <div class="container">
+          <img src="http://energycurb.com/wp-content/uploads/2015/12/curb-web-logo.png" alt="curb icon" />
+          <img src="https://s3.amazonaws.com/smartapp-icons/Partner/support/connected-device-icn%402x.png" alt="connected device icon" />
+          <img src="https://s3.amazonaws.com/smartapp-icons/Partner/support/st-logo%402x.png" alt="SmartThings logo" /> ${message}
+      </div>
+  </body>
+
+  </html>
+    """
+
+	render contentType: 'text/html', data: html
+}
+
+def getCurbClientId()        { return "R7LHLp5rRr6ktb9hhXfMaILsjwmIinKa" }
+def getCurbClientSecret()    { return "pcxoDsqCN7o_ny5KmEKJ2ci0gL5qqOSfxnzF6JIvwsfRsUVXFdD-DUc40kkhHAZR" }
```

Comment:
I suggest to move the clientId and clientSecret to `App Settings`, clientId is already defined as appSetting but not used anywhere. Either change above method to return e.g. `appSettings.clientID` or just use it in place.